just-install-v4.schema.json

```
@@ -39,6 +39,9 @@
         },
         "shims": {
           "type": "array"
+        },
+        "shortcuts": {
+          "type": "array"
```

Should add structural information about objects in the array, something like:

```suggestion
          "type": "array",
          "items": {
            "type": "object",
            "additionalProperties": false,
            "required": ["name", "target"],
            "properties": {
              "name": {
                "type": "string"
              },
              "target": {
                "type": "string"
              }
            }
          }
```